docs/detections/rules-ui-create.asciidoc

```diff
@@ -21,30 +21,42 @@ running, the rule will:
 ** Run and create alerts if existing anomaly scores above the defined threshold
 are discovered.
 ** Issue an error stating the {ml} job was not running when the rule executed.
-* *Threshold rules*: Searches the defined indices and creates a detections alert
+* <<create-threshold-rule, *Threshold rules*>>: Searches the defined indices and creates a detections alert
 when the number of times the specified field's value meets the threshold during
 a single execution. When multiple values meet the threshold, an alert is
 generated for each value.
 +
 For example, if the threshold `field` is `source.ip` and its `value` is `10`, an
 alert is generated for every source IP address that appears in at least 10 of
 the rule's search results.
+* <<create-eql-rule, *Event correlation*>>: Searches the defined indices and creates an alert when results match an
+{ref}/eql.html[Event Query Language (EQL)] query.
+* <<create-indicator-rule, *Indicator match*>>: Creates an alert when {es-sec} index field values match
+field values defined in the specified indicator index patterns. For example, you
+can create a threat index for IP addresses and use this index to create an alert
+whenever an event's `destination.ip` equals a value in the index. Threat index
+field mappings should be {ecs-ref}[ECS-compliant]. For information on creating
+{es} indices and field types, see
+{ref}/getting-started-index.html[Index some documents],
+{ref}/indices-create-index.html[Create index API] and
+{ref}/mapping-types.html[Field data types].
 
-When creating or modifying rules, you can add exceptions that prevent a rule
-from generating an alert even when its criteria are met. This is useful for
-reducing noise, such as preventing alerts from trusted processes and internal
-IP addresses. <<detections-ui-exceptions>> describes how to add exceptions to a
-rule.
+When creating or modifying rules, you can add exceptions that prevent a rule from generating an alert even when its criteria are met. This is useful for reducing noise, such as preventing alerts from trusted processes and internal IP addresses. <<detections-ui-exceptions>> describes how to add exceptions to a rule.
+
+NOTE: You can add exceptions to custom query, EQL, and indicator match rule types.
+
+If you are creating a custom query, threshold, or EQL rule, you can preview the rule beforehand to see what kind of results you can expect. See <<preview-rules, Preview your rule>> in this topic for more information.
```

Comment:
Event correlation instead of EQL here?

```suggestion
If you are creating a custom query, threshold, or event correlation rule, you can preview the rule beforehand to see what kind of results you can expect. See <<preview-rules, Preview your rule>> in this topic for more information.
```